kernel/aster-nix/src/util/random.rs

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use aster_frame::sync::Mutex;
+use rand::{rngs::StdRng, Error as RandError, RngCore};
+use spin::Once;
+
+use crate::prelude::*;
+
+static RNG: Once<Mutex<StdRng>> = Once::new();
+
+pub fn getrandom(dst: &mut [u8]) -> Result<()> {
```

Comment:
Could you add Rust docs for it? The docs should emphasize whether the random byte sequence is cryptographically secure.